src/cli/adapter/eslint.ts

```diff
@@ -53,15 +60,29 @@ export const runESLint = async (
     )}.`,
   );
 
-  let errors = 0;
+  const errors: ESLintResult[] = [];
+  const warnings: ESLintResult[] = [];
 
   for (const result of results) {
-    errors += result.errorCount;
-
-    logger.debug(symbolForResult(result), path.relative(cwd, result.filePath));
+    const relativePath = path.relative(cwd, result.filePath);
```

Comment:
👍 This looks similar to https://github.com/seek-oss/skuba/blob/5dc3a2a4492f6095aaa850507d6eb3fd20361ec0/src/cli/adapter/eslint.ts#L61

Maybe we should have our adapter calculate its own `relativeFilePath` for reuse in the linked log message and here?

Reply:
A little bit confused at this one - but yeah I rewrote it to use the relativePath const

https://github.com/seek-oss/skuba/blob/9d97f1c9bb0b38ce37fbbe9be2c382d85f486583/src/cli/adapter/eslint.ts#L82